.gitignore

```diff
@@ -39,8 +39,8 @@ webportal/webapp/.DS_Store
 webportal/webapp/Thumbs.db
 
 # Backend files
-webportal/lambda/__pycache__
-webportal/lambda/.chalice
+__pycache__
```

Comment:
If you want wildcard matching (for nested dirs or top-level), then you might want `**/__pycache__` _(Basing this assumption on the prior code)_

Reply:
This seems to work... at least for what I'm trying to do :man_shrugging: